### PR TITLE
Option 'verbose' is not available anymore

### PR DESCRIPTION
**Pull request by domhof**

### CODE REVIEW

File: docs/Actions.md [hunk 1]
```diff
@@ -100,7 +100,6 @@ ipa(
   identity: "MyIdentity",          # Identity to be used along with --embed
   sdk: "10.0",                     # use SDK as the name or path of the base SDK when building the project.
   archive: nil,                    # this means 'Do Archive'. Archive project after building.
```
Comment (KrauseFx) [inline docs/Actions.md]: Please remove the `,` at the end of the line.

**Comment by KrauseFx** [comment]
Thanks nice catch, please update the one little thing before merging :+1:

**Comment by KrauseFx** [comment]
Thank you @domhof. You know you can also get the latest documentation using `fastlane action ipa`? I plan to generate `Actions.md` completely automatic using the documentation directly from the code.

**Comment by domhof** [comment]
Cool, thanks for the hint! :+1:

**Pull request merged**